bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/ghost/GhostMixin.java

```diff
@@ -0,0 +1,10 @@
+package edu.gemini.spModel.gemini.ghost;
+
+import edu.gemini.pot.sp.SPComponentType;
+
+// We need to do this because of reflection. This is a static property of instruments, but since Ghost is
```

Comment:
Does it make any sense attempting to remove reflection?

Reply:
I would love to see reflection go... I'm not sure how difficult that would be to eliminate in this case since `UIInfo.xml` contains a good amount of data. @swalker2m? What do you think?

Reply:
Agree that reflection is bad but I don't think we want to take that on at this point.